packages/SwingSet/test/test-gc-and-finalize.js

```diff
@@ -81,7 +76,8 @@ test(`can provoke gc on xsnap`, async t => {
   const opts = options();
   const vat = xsnap(opts);
   const code = `
-${gcAndFinalize}
+${makeGcAndFinalize}
+const engineGC = globalThis.gc;
```

Comment:
It seems somewhat ironic that we must extract `gc` from the global after going through so much work to avoid putting it on the global. Although I suppose the main benefit of using the `expose-gc` NPM module is that we don't need command-line flags, rather than reducing authority added to the start-compartment global.

Reply:
No irony.  On **Node.js**, `import engineGC from 'expose-gc/function';` does not modify `globalThis`.

The code above is evaluated under **xsnap**, to which the C code has deliberately endowed `globalThis.gc`.

Reply:
oh, right, I assumed their `function.js` was only reachable through their `index.js`, but of course you can deep-import it directly